source/catalog.html

```diff
@@ -0,0 +1,254 @@
+              <h3 class="product__card-title">Cat Energy SLIM 1000 г</h3>
+            </a>
+            <d1 class="product__specifications">
+              <dt class="product__specifications-definition">Масса</dt>
+              <dd class="product__specifications-meaning">1000 г</dd>
+              <dt class="product__specifications-definition">Вкус</dt>
+              <dd class="product__specifications-meaning">Гречка</dd>
+              <dt class="product__specifications-definition">Цена</dt>
+              <dd class="product__specifications-meaning">500 Р.</dd>
+              <a href="#" class="button product__button">заказать</a>
+          </li>
+          <li class="product__card">
+            <a class="product__image-link" href="#">
+              <img class="product__image" src="" width="72" height="84" alt="500gr">
+            </a>
+            <a class="product__title" href="#">
+              <h3 class="product__card-title">Cat Energy SLIM 500 г</h3>
+            </a>
+            <d1 class="product__specifications">
+              <dt class="product__specifications-definition">Масса</dt>
+              <dd class="product__specifications-meaning">500 г</dd>
+              <dt class="product__specifications-definition">Вкус</dt>
+              <dd class="product__specifications-meaning">Рис</dd>
+              <dt class="product__specifications-definition">Цена</dt>
+              <dd class="product__specifications-meaning">500 Р.</dd>
+              <a href="#" class="button product__button">заказать</a>
+          </li>
+          <li class="product__card product__card--pagination">
+            <a class="product__card product__card--pagination-link" href="#">
+              <p class="product__card-text">Показать еще 100500 товаров</p>
+              <p class="product__card-description">На самом деле вкусов гораздо больше!</p>
+            </a>
+            <button class="button product__card-button" type="button">Показать все</button>
+          </li>
+        </ul>
+      </div>
+    </section>
+
+    <section class="additional">
+      <h2 class="additional__title">Дополнительные товары</h2>
+      <ul class="additional__list">
+        <li class="additional__title">
+          <a class="additional__link" href="#">
+            Сахарозаменитель
+            <p class="additional__number">1 упаковка (100 г)</p>
+            <p class="additional__sum">200 Р.</p>
+          </a>
+          <button class="button additional__button" type="button">заказать</button>
+        </li>
+        <li class="additional__title">
+          <a class="additional__link" href="#">
+            питьевая вода
+            <p class="additional__number">5 литров</p>
+            <p class="additional__sum">50 Р.</p>
+          </a>
+          <button class="button additional__button" type="button">заказать</button>
+        </li>
+        <li class="additional__title">
+          <a class="additional__link" href="#">
+            молоко
```

Comment:
Тут тоже хотелось бы обычных текстов, как мы пишем всегда

```suggestion
            Молоко
```

Reply:
ага, везде вроде поправила регистр